fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/BadRequestExceptionMapper.java

```diff
@@ -44,7 +45,7 @@
     public Response toResponse(final BadRequestException e) {
         LOGGER.error("BadRequestExceptionMapper caught an exception: {}", e.getMessage());
         debugException(this, e, LOGGER);
-        return status(BAD_REQUEST).entity(e.getMessage()).build();
+        return status(BAD_REQUEST).entity(e.getMessage()).type(DEFAULT_CONTENT_TYPE).build();
```

Comment:
I think the clarity of the code would be increased by replacing `DEFAULT_CONTENT_TYPE` with the actual "TEXT_PLAIN_WITH_CHARSET".
I realize that by abstracting the actual content-type with a global variable allows for changing the value of that variable in the future... but in that theoretical scenario, there is an equally good chance that these classes will change as well and my sense would be the same then, too; namely, that seeing the actual content-type is more clear.